dbt/parser.py

```diff
@@ -218,6 +218,11 @@ def parse_node(node, node_path, root_project_config, package_project_config,
     config_dict.update(config.config)
     node['config'] = config_dict
 
+    # Set this temporarily so get_rendered() below has access to a schema
+    profile = dbt.utils.get_profile_from_project(root_project_config)
+    default_schema = profile.get('schema', 'public')
+    node['schema'] = default_schema
```

Comment:
couldn't this render in the wrong schema for multi-schema setups?

Reply:
yeah, it might, but that would only happen during parsing, so it shouldn't have any effect. If other functions come along in the future, they'll get a somewhat sane value for a schema instead of `None`.